Vagrantfile

```diff
@@ -9,7 +9,7 @@ Vagrant.configure('2') do |config|
     vb.customize ['modifyvm', :id, '--ioapic', 'on']
   end
 
-  config.vm.define :vagrant_demo_server do |service_config|
+  config.vm.define :rr_provision do |service_config|
```

Comment:
I have turned back the name of a machine. Moving forward and merge it for now as is.